aws-sdk-go/leo.go

```diff
@@ -0,0 +1,252 @@
+package main
+
+import (
+        "log"
+        "io/ioutil"
+        "bytes"
+        "crypto/rand"
+        "os"
+
+        "github.com/aws/aws-sdk-go/aws"
+        "github.com/aws/aws-sdk-go/aws/awsutil"
+        "github.com/aws/aws-sdk-go/service/s3"
+        "github.com/aws/aws-sdk-go/aws/credentials"
+       )
+
+const (
+        BucketName = "testgo"
+        FileName   = "testFile"
+        LargeFileName   = "testFile.large"
+        DataPath   = "../temp_data/"
+        ChunkSize  = 5 << 20
+        LargeObjSize = 50 << 20
+      )
+
+
+func main() {
+    cred := credentials.NewStaticCredentials("05236", "802562235", "")
+
+    svc := s3.New(&aws.Config{Endpoint: "http://localhost:8080",
+                              Region: "us-west-2",
+                              Credentials: cred,
+                              LogLevel: 0})
+
+    bucketStr := aws.String(BucketName)
+
+    _, err := svc.CreateBucket(&s3.CreateBucketInput{Bucket: bucketStr})
+
+    if err != nil {
+        log.Printf("Create Bucket %s Failed, %s\n", BucketName, err)
```

Comment:
https://golang.org/pkg/log/#Fatalf is more suitable when exiting with logging.
please apply to other places.